core/src/components/reorder-group/reorder-group.tsx

```diff
@@ -55,7 +56,9 @@ export class ReorderGroup implements ComponentInterface {
   async connectedCallback() {
     const contentEl = this.el.closest('ion-content');
     if (contentEl) {
-      this.scrollEl = await contentEl.getScrollElement();
+      componentOnReady(contentEl, async () => {
+        this.scrollEl = await contentEl.getScrollElement();
+      });
```

Comment:
Are we able to do all of the setup inside of the `componentOnReady` callback? Otherwise it's possible that the gesture could get setup prior to `this.scrollEl` being set.

Reply:
I can do that, or I can promisefy the `componentOnReady` and `await` it. 

But is that necessary? The gesture is referencing the `this.scrollEl` value not during setup, but rather when the gesture occurs by calling a class method: `canStart: detail => this.canStart(detail)`

So it should always be up to date, right?

Reply:
I don't have any strong preference, so I will leave it up to you. However, the way this PR does it would be a behavior change from how it used to work. Previously, the gesture would not be setup until the scroll element was set. Whereas now the gesture _could_ be setup before the scroll element was set.

Given that the scroll element is referenced in the gesture callbacks, this could have some negative effects. I don't think this is likely, but it depends on how safe we want to play it.